ROADMAP.md

```diff
@@ -0,0 +1,14 @@
+# Skopeo Roadmap
+
+The Skopeo development team reviews feature requests from its various stakeholders for consideration
+quarterly.  Skopeo maintainers then prioritize these features.   Top features are then assigned to
+one or more engineers.
+
+
+## Future feature focus
+
+* OCI artifact support
+* Integration of composefs
+* Partial pull support (zstd:chunked).
+* Performance and stability improvements.
```

Comment:
These are all c/image and lower.

Now, is this a roadmap for “Skopeo the delivered binary”? If so, listing things that would be better just by updating other projects is fine.

But if this is a roadmap for contributors, we ~don’t want contributions of that kind in _this_ repo.

I think it’s more of the latter?

Reply:
yes I know, and I am dealing with it as with any bureaucratic piece of paper I have to deal with occasionally.  Let's just write something :-)

Reply:
I think I’m fine with that for process/bureaucracy (e.g. I really don’t care about “assigning engineers” in this document), but in case we did get people interested in newly contributing, I’d prefer to direct them to more productive efforts like c/image.

And if the roadmap looks ~empty, I think that’s fine too.